README.md

```diff
@@ -35,36 +35,3 @@ When you need to create new pnc-api-types-ts release from unreleased PNC revisio
 You need to clone repository and run release.sh script with desired **PNC** revision, after script finished you can check the result and push new release into remote repository.
 
 for example: `./release.sh 2.3.0` or `./release.sh 0bf26885ec147b815be8be6f16cf484c976364c9`
- 
-#### Option D - completely manual process
```

Comment:
Also line 5 in this file (it mentions using `generate-openapi-spec.sh`) needs to be updated.